sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/MigrationGuide.md

```diff
@@ -1,120 +1,78 @@
-# Migrate from Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker to Azure.AI.Language.QuestionAnswering
+# Migrate from Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker to Azure.AI.Language.QuestionAnswering Authoring
 
-This guide is intended to assist in the migration to the new Question Answering client library [`Azure.AI.Language.QuestionAnswering`](https://www.nuget.org/packages/Azure.AI.Language.QuestionAnswering) from the old one [`Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker`](https://www.nuget.org/packages/Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker). It will focus on side-by-side comparisons for similar operations between the two packages.
+> [!NOTE]
+> Scope: This guide covers **authoring (project / knowledge sources / QnA / export / delete)** migration from QnAMaker to the modern Authoring SDK.
+> For **runtime querying (inference)** migration, see the separate runtime migration guide and the inference package `Azure.AI.Language.QuestionAnswering.Inference`.
+> Service API version referenced by current previews: `2025-05-15-preview`.
 
-Familiarity with the `Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker` library is assumed. For those new to the Question Answering client library for .NET, please refer to the [`Azure.AI.Language.QuestionAnswering` README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md) and [`Azure.AI.Language.QuestionAnswering` samples](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/samples) for the `Azure.AI.Language.QuestionAnswering` library rather than this guide.
+This guide helps migrate from the old client library [`Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker`](https://www.nuget.org/packages/Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker) to the modern authoring client library `Azure.AI.Language.QuestionAnswering.Authoring`. It focuses on side‑by‑side comparisons for authoring operations (creating/updating/exporting/deleting).
+
+For general usage and more samples, refer to the [Authoring README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md) and [Authoring samples](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/samples).
 
 ## Table of contents
 
 - [Migration benefits](#migration-benefits)
 - [General changes](#general-changes)
   - [Package and namespaces](#package-and-namespaces)
-  - [Runtime Client](#runtime-client)
-    - [Authenticating runtime client](#authenticating-runtime-client)
-    - [Querying a question](#querying-a-question)
-    - [Chatting](#chatting)
-  - [Authoring Client](#authoring-client)
+  - [Package split and choosing the right package](#package-split-and-choosing-the-right-package)
+  - [Type forwarding & compatibility](#type-forwarding--compatibility)
```

Comment:
"Type forwarding & compatibility" is not the migration way we want to push forwards. This is only a backwards compatibility step taken only for this package. Do we need this in the migration guide?